ophyd/pseudopos.py

```diff
@@ -126,11 +334,14 @@ class PseudoPositioner(Device, Positioner):
     '''
     def __init__(self, prefix, *, concurrent=True, read_attrs=None,
                  configuration_attrs=None, monitor_attrs=None, name=None,
-                 **kwargs):
+                 egu='', **kwargs):
```

Comment:
Yeah, that's what I had in mind. And, is there an example where PseudoSingle and its PseudoPositioner would disagree about the units?